scripts/build.sh

```
@@ -7,29 +7,30 @@
+cp target/release/zrc "$ZIRCON_TOOLCHAIN_DIR/bin"
+cp target/release/zircop "$ZIRCON_TOOLCHAIN_DIR/bin"
+cp target/release/zrx "$ZIRCON_TOOLCHAIN_DIR/bin"
+cp target/release/zrepl "$ZIRCON_TOOLCHAIN_DIR/bin"
+cp -r include/* "$ZIRCON_TOOLCHAIN_DIR/include/zirco/"
+cp target/release/libzrc.a "$ZIRCON_TOOLCHAIN_DIR/lib/"
+find target/release -maxdepth 1 -type f \( -name "libzrc.so" -o -name "libzrc.dylib" \) -exec cp {} "$ZIRCON_TOOLCHAIN_DIR/lib/" \;
+cp compiler/zrc_c/zrc.h "$ZIRCON_TOOLCHAIN_DIR/include/"
 
 # build std using the fresh compiler
-make -C libzr all-opt ZRC=$ZIRCON_BIN_DIR/zrc
+make -C libzr all-opt ZRC=$ZIRCON_TOOLCHAIN_DIR/bin/zrc
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)